bgpd/bgp_attr.c

```diff
@@ -2927,9 +2927,21 @@ bgp_attr_psid_sub(uint8_t type, uint16_t length,
 	int srgb_count;
 	uint8_t sid_type, sid_flags;
 
+	/*
+	 * Check that we actually have at least as much data as
+	 * specified by the length field
+	 */
+	if (STREAM_READABLE(peer->curr) < length) {
+		flog_err(
+			EC_BGP_ATTR_LEN,
+			"Prefix SID Originator SRGB specifies length %hu, but only %zu bytes remain",
```

Comment:
Isn't this a generic error message, not only Originator SRGB-specific?

Reply:
I'm not sure what you are asking/implying here.  It's a specific error message to when we are parsing the Prefix SID hence the message that way

Reply:
I mean this error message is generic and valid not only for BGP_PREFIX_SID_ORIGINATOR_SRGB, but for BGP_PREFIX_SID_IPV6 or any other type?